conf.sh

```diff
@@ -0,0 +1,2 @@
+export TF_VAR_route53_zone_name="bda.ninja"
+export TF_VAR_google_oidc_application="274210205449-bend56f57m2gcu9an3q6dmv0atj3i7h1.apps.googleusercontent.com"
```

Comment:
Is it really ok to version this ? on the global internets ?

Reply:
Yes, this is just an id pointing to the Google application towards which we authenticate. It would typically be set in the js on client side so that the client knows on which app it should query its identity. And in the end, we will set a different one in our firm account.